src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -68,7 +69,10 @@ public class PropertiesMacro extends AbstractMacro {
     /** A newline with 20 spaces of indentation. */
     private static final String INDENT_LEVEL_20 = SiteUtil.getNewlineAndIndentSpaces(20);
 
-    /** The name of the current module being processed. */
+    /** The property being process with all tokens. */
+    private static final String TOKENS_PROPERTY = SiteUtil.TOKENS;
+
+    /** The file of the current module being processed. */
```

Comment:
Why is this changed?

Reply:
@nrmancuso This property is used to correct the ordered properties.

Reply:
It might looks now different than it was before.
But now I see no difference, and GitHub shows diff with extra lines change but content is not changed

Reply:
Added @romani

Reply:
Why did we change `name` to `file`? I would expect something that is called a `file` to be of type `File`. Please revert this change.

Reply:
@AayushSaini101, please restore this line as it was before.

Reply:
@nrmancuso @romani Done